addons/killtracker/initSettings.inc.sqf

```diff
@@ -0,0 +1,10 @@
+[
+    QGVAR(trackAI),
+    "CHECKBOX",
+    [LSTRING(TrackAI_DisplayName), LSTRING(TrackAI_Description)],
+    LSTRING(Category),
+    true,
+    true,
+    {},
+    true
```

Comment:
Why does this require a mission start?
The setting takes effect immediately, although I understand that it will omit any AI kills during the time it's off.

Reply:
Yeah, shouldn't require a reboot cuz if the Players care enough to have a total accuracy (or don't wanna take any chances), they'll reboot, otherwise they'll accept the imprecision.